src/go/rdctl/cmd/snapshotCreate.go

```diff
@@ -12,21 +14,33 @@ var snapshotCreateCmd = &cobra.Command{
 	Use:   "create <name>",
 	Short: "Create a snapshot",
 	Args:  cobra.ExactArgs(1),
-	RunE:  wrapSnapshotOperation(createSnapshot),
+	RunE: func(cmd *cobra.Command, args []string) error {
+		cmd.SilenceUsage = true
+		err := wrapSnapshotOperation(createSnapshot)(cmd, args)
+		return exitWithJsonOrErrorCondition(err)
+	},
 }
 
 func init() {
 	snapshotCmd.AddCommand(snapshotCreateCmd)
+	snapshotCreateCmd.Flags().BoolVarP(&outputJsonFormat, "json", "", false, "output json format")
 }
 
-func createSnapshot(cmd *cobra.Command, args []string) error {
-	paths, err := p.GetPaths()
+func createSnapshot(_ *cobra.Command, args []string) error {
+	appPaths, err := paths.GetPaths()
```

Comment:
I'd rather not, because single-letter variable names are harder to search and modify. Loop indices are one thing, but I don't see the problem with `appPaths`

Reply:
This is somewhat inconsistent with your willingness to import the package under the name `p`. Shorter variable names should be restricted to smaller lexical scopes.

That said, I have no problem with using `appPaths`, as long as this is done consistently. We just have to pick one name and use it everywhere.